Common/Inc/assert.h

```diff
@@ -0,0 +1,6 @@
+#ifndef ASSERT_H
```

Comment:
Use a different file name, assert is part of the standard library and we don't want to cause confusion between C standard assert.h and our own assert.